### PR TITLE
use pdp_retention_interval

### PR DESCRIPTION
**Pull request by ghost**
pdp_retention_interval is set in init.pp, but not used.

### CODE REVIEW

File: templates/pdp.ini.erb [hunk 1]
```diff
@@ -30,7 +30,7 @@ adminPassword = pdpadmin_<%= @facts['uniqueid'] %>
 
 [POLICY]
 paps = https://<%= @argus_host %>:8150/pap/services/ProvisioningService
-retentionInterval = 240
+retentionInterval = <%= @pdp_retention_interval %>
```
Comment (kreczko) [inline templates/pdp.ini.erb:33]: This also adds a new line to the config file.
Could you please change it to `<%= @pdp_retention_interval -%>` ?

Comment (ghost) [reply templates/pdp.ini.erb:33]: It is not a script line therefore it should be `%>` instead of `-%>` as same as other variables.

**Comment by ghost** [comment]
It needs another change in config.pp to follow up the last commit.
I close this and make another PL.

File: templates/pdp.ini.erb (hunk 1 above)
Comment (kreczko) [reply templates/pdp.ini.erb:33]: my bad, thanks

**Pull request closed without merging**